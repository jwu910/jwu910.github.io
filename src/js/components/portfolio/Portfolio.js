import React from 'react';

import ProjectList from './ProjectList'

class Portfolio extends React.Component {
  render() {
    return (
      <div>
        <div className="portfolio-wrapper">
          <ProjectList />
        </div>
      </div>
    );
  }
}

export default Portfolio;