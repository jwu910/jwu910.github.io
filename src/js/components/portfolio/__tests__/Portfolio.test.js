import React from 'react';
import renderer from 'react-test-renderer';

import Portfolio from '../Portfolio';

describe('Snapshots', () => {
  it('Renders correctly', () => {
    const component = renderer
      .create(
        <Portfolio />
      )
      .toJSON();

    expect(component).toMatchSnapshot();
  });
});