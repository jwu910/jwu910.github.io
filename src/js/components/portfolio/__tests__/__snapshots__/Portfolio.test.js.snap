// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshots Renders correctly 1`] = `
<div>
  <div
    className="portfolio-container"
  >
    <div
      className="project-list-container"
    >
      <ul>
        <li>
          <div
            className="project-card-container"
          >
            <a
              className="githubLink"
              href="https://github.com/jwu910/check-it-out"
            >
              <div
                className="card-wrapper"
              >
                <div
                  className="card-box"
                >
                  <div
                    className="card-title"
                  >
                    <span
                      className="title"
                    >
                      Check It Out
                    </span>
                  </div>
                  <div
                    className="card-content"
                  />
                </div>
              </div>
            </a>
          </div>
        </li>
        <li>
          <div
            className="project-card-container"
          >
            <a
              className="githubLink"
              href="https://github.com/jwu910/branchnote"
            >
              <div
                className="card-wrapper"
              >
                <div
                  className="card-box"
                >
                  <div
                    className="card-title"
                  >
                    <span
                      className="title"
                    >
                      Branchnote
                    </span>
                  </div>
                  <div
                    className="card-content"
                  />
                </div>
              </div>
            </a>
          </div>
        </li>
        <li>
          <div
            className="project-card-container"
          >
            <a
              className="githubLink"
              href="https://github.com/5-gwoap/slng-node"
            >
              <div
                className="card-wrapper"
              >
                <div
                  className="card-box"
                >
                  <div
                    className="card-title"
                  >
                    <span
                      className="title"
                    >
                      Slng
                    </span>
                  </div>
                  <div
                    className="card-content"
                  />
                </div>
              </div>
            </a>
          </div>
        </li>
      </ul>
    </div>
  </div>
</div>
`;
